rbac/api/common/exception_handler.py

```diff
@@ -82,4 +83,21 @@ def custom_exception_handler(exc, context):
             },  # noqa: E231
             status=status.HTTP_400_BAD_REQUEST,
         )
+    elif isinstance(exc, MissingAuthorizationError):
+        source_view = context.get("view")
+        response = Response(
+            data={
+                "errors": [
+                    {
+                        "detail": "A Bearer token in an authorization header is required when"
+                        " performing service account operations.",
+                        "source": source_view.basename,
+                        "status": str(status.HTTP_401_UNAUTHORIZED),
+                    }
+                ]
+            },
+            content_type="application/json",
+            status=status.HTTP_401_UNAUTHORIZED,
+        )
+
```

Comment:
I considered the following alternatives:

- Throwing the `MissingAuthorizationError` in the ITService class, but handle the exception manually on every view that it gets instantiated and used.
- Before calling any ITService methods perform an `if not user.bearer_token` check and return a 403 response.

However it felt like it was just better to handle the exception in the custom exception handler we had. What do you think?

Reply:
👍  this makes sense to me - going to look into why the PR checks are failing 👀

Reply:
All right, I'm going to resolve this conversation, then!